src/tools/compiletest/src/runtest.rs

```diff
@@ -2175,6 +2175,7 @@ impl<'test> TestCx<'test> {
                     "-Zvalidate-mir",
                     "-Zlint-mir",
                     "-Zdump-mir-exclude-pass-number",
+                    "-Zmir-include-spans=false",
```

Comment:
I'm not totally certain I understand the consequence of this.

Reply:
Just to avoid printing spans in the nll tests? Should probably have a comment specifying that if so.

Reply:
That's right, I've mentioned this in the PR and commit message, but have also now added a comment.